pkg/types/profit.go

```diff
@@ -194,12 +194,12 @@ func (s *ProfitStats) AddProfit(profit Profit) {
 	s.TodayPnL = s.TodayPnL.Add(profit.Profit)
 	s.TodayNetProfit = s.TodayNetProfit.Add(profit.NetProfit)
 
-	if profit.Profit.Sign() < 0 {
+	if profit.Profit.Sign() > 0 {
```

Comment:
any reason why we make the change on this file? the result seems the same.

Reply:
To make it more intuitive to ready, process the positive profit first.